src/EditorFeatures/Core/EditorConfigSettings/Updater/SettingsUpdateHelper.cs

```diff
@@ -4,7 +4,9 @@
 
 using System;
 using System.Collections.Generic;
+using System.Collections.Immutable;
```

Comment:
I think we need to delete this file or merge it with [ConfigurationUpdater](https://github.com/dotnet/roslyn/blob/main/src/Features/Core/Portable/CodeFixes/Configuration/ConfigurationUpdater.cs) as soon as possible. `ConfigurationUpdater` performs the exact same functionality as the this helper type, and is used by the `Configure Severity` and `Configure Code Style` light bulb actions. Having this functionality duplicated makes it very to maintain and review. Do we plan to do this merge in Dev16 or Dev17?

Reply:
@jinujoe triaged it into 16.11 https://github.com/dotnet/roslyn/issues/52735. My initial plan was to unify these types in this PR but we triaged the bugs into different releases and we lock down Wed.

Reply:
Thanks, sounds good to me!